src/libMessage/Messenger.cpp

```diff
@@ -145,6 +145,55 @@ void ShardToProtoCommittee(const Shard& shard, ProtoCommittee& protoCommittee) {
   }
 }
 
+void BlockBaseToProtobuf(const BlockBase& base,
+                         ProtoBlockBase& protoBlockBase) {
+  // Block hash
+
+  protoBlockBase.set_blockhash(base.GetBlockHash().data(),
+                               base.GetBlockHash().size);
+  // Serialize cosigs
+
+  ZilliqaMessage::ProtoBlockBase::CoSignatures* cosigs =
+      protoBlockBase.mutable_cosigs();
+
+  SerializableToProtobufByteArray(base.GetCS1(), *cosigs->mutable_cs1());
+  for (const auto& i : base.GetB1()) {
+    cosigs->add_b1(i);
+  }
+  SerializableToProtobufByteArray(base.GetCS2(), *cosigs->mutable_cs2());
+  for (const auto& i : base.GetB2()) {
+    cosigs->add_b2(i);
+  }
+}
+
+void ProtobufToBlockBase(BlockBase& base,
```

Comment:
Can you reverse the order of the parameters, for consistency with the other functions